src/migrations/20200228144719-product.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    const tableFields = {
```

Comment:
This variable is not needed, you can simply execute the createTable command with the fields data in it.

Reply:
Got it!